src/stores/GuildChannelStore.js

```diff
@@ -59,6 +52,7 @@ class GuildChannelStore extends DataStore {
    *   ],
    * })
    */
+
```

Comment:
I don't think there should be empty lines between the jsdocs for a method and the method itself.